### ISSUE TITLE
Motivation for Transfer Learning

### ISSUE THREAD

**Issue by iam-pattan** [HydPy/HydPy-meetups#31]
**Motivation for Transfer Learning**

**Abstract:** Traditionally, we train our deep learning models with lots of training data for n-epochs and wait for hours, days or even months, but still we won't be able to achieve the high accuracy or best scores for the model to be deployed in the real world applications. By understanding transfer learning techniques and the different types of Pre-trained models, their basic architectures which help in training models. And when deployed how they're going to be trained just for a few epochs can achieve high scores.

**Category of the talk:** Data Science, Machine Learning and AI(Deep Learning)

**Duration:** 30-45 minutes

**Level of Audience:** Intermediate

**Speaker** Bio:
Pattan Afrid Ahmed
M.tech in CSE with Spl. in Artificial Intelligence and Machine Learning
Vellore Institute of Technology, Chennai.
ahmed.afrid@gmail.com
Student(Pursuing)

**Prerequisites:**
Just a google account for Colab to work with.

**Comment by GoelJatin** [comment]
Hey @iam-pattan , apologies for the delay, the team has been occupied with some other commitments the past couple of months.

We'll review this proposal and get back to you ASAP and plan a meet-up.

**Comment by iam-pattan** [comment]
Okay.
No worries

**Comment by ananyo2012** [comment]
Talk is scheduled for tomorrow - 28th Jan at 4 pm - https://www.meetup.com/HydPyGroup/events/283237237/
